### PR TITLE
fix: bell icon and notification badge [#1470]

### PR DESCRIPTION
**Pull request by konradodo**

### CODE REVIEW

File: src/popup/components/Header.vue [hunk 1]
```diff
@@ -312,17 +314,19 @@ export default {
...
       height: 14px;
-      background: variables.$color-blue;
-      border-radius: 50%;
+      background: variables.$color-pink;
+      border-radius: 7px;
       text-align: center;
-
-      @extend %face-sans-12-regular;
+      line-height: 14px;
+      padding: 0 3.5px;
```
Comment (pstachel) [inline src/popup/components/Header.vue:329]: I think there is no thing as half of a pixel

**Review by pstachel — CHANGES_REQUESTED** [review]
The icon shouldn't have width & height as well as the color should be set to `currentColor`

File: src/popup/components/Header.vue [hunk 2]
```diff
@@ -312,17 +314,19 @@ export default {
     position: relative;
 
     .badge {
```
Comment (peronczyk) [inline src/popup/components/Header.vue:316]: I think we should have separate component for the badges.

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/popup/components/Header.vue [hunk 3]
```diff
@@ -120,9 +120,11 @@ export default {
       return this.$route.meta.hideNotificationsIcon;
     },
     notificationsCount() {
-      return [...this.notifications, ...this.superheroNotifications].filter(
+      const count = [...this.notifications, ...this.superheroNotifications].filter(
```
Comment (pstachel) [inline src/popup/components/Header.vue:123]: `this.notifications` and `this.superheroNotifications` may be undefined

**Review by pstachel — CHANGES_REQUESTED** [review]

**Review by pstachel — APPROVED** [review]

**Review by peronczyk — APPROVED** [review]

**Pull request merged**